src/overlays/actors/ovl_Boss_Fd2/z_boss_fd2.h

```diff
@@ -6,11 +6,73 @@
 
 struct BossFd2;
 
+typedef void (*BossFd2ActionFunc)(struct BossFd2*, GlobalContext*);
+
+typedef struct BossFd2Mane {
+    /* 0x000 */ Vec3f rot[10];
+    /* 0x078 */ Vec3f pos[10];
+    /* 0x0F0 */ Vec3f pull[10];
+    /* 0x168 */ f32 scale[10];
+    /* 0x190 */ Vec3f head;
+} BossFd2Mane; //size = 0x19C
+    
+
 typedef struct BossFd2 {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1530];
+    /* 0x014C */ SkelAnime skelAnime1;
```

Comment:
why the 1? there are more? same as previous collider1 and this has collider1 too